internal/cli/command/bucket/delete.go

```diff
@@ -0,0 +1,134 @@
+// Copyright © 2018 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bucket
+
+import (
+	"context"
+
+	"github.com/antihax/optional"
+	"github.com/goph/emperror"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"gopkg.in/AlecAivazis/survey.v1"
+
+	"github.com/banzaicloud/banzai-cli/.gen/pipeline"
+	"github.com/banzaicloud/banzai-cli/internal/cli"
+	"github.com/banzaicloud/banzai-cli/internal/cli/format"
+	"github.com/banzaicloud/banzai-cli/internal/cli/input"
+	"github.com/banzaicloud/banzai-cli/internal/cli/utils"
+)
+
+type deleteBucketsOptions struct {
+	name           string
+	cloud          string
+	location       string
+	storageAccount string
+}
+
+// NewDeleteCommand creates a new cobra.Command for `banzai bucket delete`.
+func NewDeleteCommand(banzaiCli cli.Cli) *cobra.Command {
+	o := deleteBucketsOptions{}
+
+	cmd := &cobra.Command{
+		Use:     "delete",
+		Short:   "Delete bucket",
+		Long:    "Delete Pipeline managed object storage bucket",
```

Comment:
pls mention here whether this removes the bucket from the cloud provider or just pipeline

Reply:
👍